### ISSUE TITLE
Keyboard shortcuts for selecting/deselecting files for bulk actions

### ISSUE THREAD

**Issue by clarkepaul** [silverstripe/silverstripe-admin#271]
It would be good to have keyboard shortcuts which are native to PC or Mac (eg. shift key and command key for Mac) for selecting multiple files. 

* A user should be able to select the first file and last item so that all files between them are selected
* A user should be able to select the last file and then the first item so that all files between them are selected
* A user should be able to use the sort functionality and still be able to make a selection
* Folders should be able to include in the selection

**Comment by clarkepaul** [comment]
@scott1702 can you confirm the above points with your understanding (even if you don't submit what you've done already).

**Comment by scott1702** [comment]
Yep that functionality is working here https://github.com/silverstripe/silverstripe-asset-admin/pull/633. Just need to add some tests and fix a minor bug.

**Comment by clarkepaul** [comment]
Related https://github.com/silverstripe/silverstripe-admin/issues/1240
